Daf.Meta/Layers/DataSource.cs

```diff
@@ -387,20 +369,31 @@ public Build Build
 			return Model.Instance.GetBusinessKey(BusinessKey.Name);
 		}
 
-		[Browsable(false)]
-		public ObservableCollection<HubRelationship> HubRelationships { get; } = new ObservableCollection<HubRelationship>();
+		public ObservableCollection<HubRelationship> HubRelationships { get; } = new();
+
```

Comment:
Inconsistent white space, probably best to remove the empty line.

Reply:
Removed linebreaks between consecutive property declarations which are on a single line.